test/WebJobs.Script.Tests.Integration/Host/WebJobsScriptHostServiceTests.cs

```diff
@@ -199,9 +199,6 @@ public void IsHostHealthy_ReturnsExpectedResult()
 
             var ex = Assert.Throws<InvalidOperationException>(() => _scriptHostService.IsHostHealthy(true));
             Assert.Equal("Host thresholds exceeded: [Foo, Bar]. For more information, see https://aka.ms/functions-thresholds.", ex.Message);
-
-            _healthMonitorOptions.Enabled = false;
```

Comment:
Why the deletion here?

Reply:
There was some weird merge issue that I messed up. I noticed this as well and have backed out the changes to this file.